cluster/stone-prd-rh01/managed/rhtap-releng-tenant/remote-secrets/remote_secret-quay-token-redhat-acm.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: RemoteSecret
+metadata:
+  name: quay-token-secret-remote-secret
```

I think you need a better name here. 

```suggestion
  name: quay-token-redhat-acm-remote-secret
```

---

Fixed, thanks!